Carrot-Assistant/components/embeddings.py

```diff
@@ -0,0 +1,96 @@
+from enum import Enum
+from urllib.parse import quote_plus
+from dotenv import load_dotenv
+from haystack.dataclasses import Document
+from haystack_integrations.document_stores.qdrant import QdrantDocumentStore
+from haystack_integrations.components.retrievers.qdrant import QdrantEmbeddingRetriever
+from haystack_integrations.components.embedders.fastembed import FastembedDocumentEmbedder, FastembedTextEmbedder
+import os
+from os import environ
+from sqlalchemy.orm import Session
+from sqlalchemy import create_engine
+from typing import List
+
+from omop.omop_models import Concept
+
+class EmbeddingModel(Enum):
```

Comment:
- Why are we using two models? 

- Are we going to test 2 models and choose the best one? 



Could you please add the docstring comment below: 

`class EmbeddingModel(Enum):
    """
    This Enum class contains the available embedding models and their dimensions.
    
    The Enum class contains the following models:
    
    models
    -------
    
    BGESMALL: str
        The BAAI/bge-small-en-v1.5 model with 384 dimensions.
        
    MINILM: str
        The sentence-transformers/all-MiniLM-L6-v2 model with 384 dimensions.
    """
    BGESMALL = "BAAI/bge-small-en-v1.5", 384
    MINILM = "sentence-transformers/all-MiniLM-L6-v2", 384

    def __init__(self, path: str, dimensions: int) -> None:
        """
        This method initializes the EmbeddingModel class.
        
        Parameters
        ----------
        self: EmbeddingModel
            The EmbeddingModel class.
            
        path: str
            The path to the model.
            
        dimensions: int
            The number of dimensions in the model.
        """
        self.path = path
        self.dimensions = dimensions`